src/server/csp.ts

```diff
@@ -137,6 +137,9 @@ function cspEditions(
 			? 'https://platform.twitter.com https://cdn.syndication.twimg.com'
 			: ''
 	};
+	frame-src https://www.theguardian.com https://www.scribd.com ${
```

Comment:
maybe remove scribd.com until we know that we need it?!

Reply:
removed :)